app/scripts/controllers/tree/tree-controller.js

```diff
@@ -2,11 +2,41 @@
   'use strict';
 
   angular.module('moi.controllers')
-  .controller('TreeController', function (data) {
+  .controller('TreeController', function ($scope, $rootScope, data, PreloadAssets, VIDEOS) {
     var treeModel = this;
     treeModel.neurons = data.tree;
     treeModel.meta = data.meta;
     treeModel.isBasicLevel = data.meta.depth < 5;
+    var videos = VIDEOS.paths;
+    var vinetaLevels = [1, 4, 6, 8];
+    var preloadMovies = false;
+    initVineta();
+
+    treeModel.finishedAnimation = function() {
+      $scope.$apply(function(){treeModel.showTree = true;});
+      localStorage.setItem('vinetas_animadas',JSON.stringify({'depth': data.meta.depth}));
+    };
+
+    function initVineta() {
+      var getConfigVineta = JSON.parse(localStorage.getItem('vinetas_animadas'));
```

Comment:
ño, han creado un service con ese nombre https://forum.ionicframework.com/t/store-objects-in-localstorage/44788/3